app/channels/synchronized_editor_channel.rb

```diff
@@ -3,11 +3,13 @@
 class SynchronizedEditorChannel < ApplicationCable::Channel
   def subscribed
     stream_from specific_channel
+    ActionCable.server.broadcast(specific_channel, {command: 'connection_change', status: 'connected', current_user_id: current_user.id, current_user_name: current_user.name})
```

Comment:
Here, and also below, we should potentially use the user_type as well (if this change is still present after switching to a UUID.